### PR TITLE
Fix isQuantumReady definition in main_summary

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala [hunk 1]
```diff
@@ -215,42 +215,57 @@ object MainSummaryView {
...
       "firefox-compact-light@mozilla.org",
       "firefox-compact-dark@mozilla.org"
     )
 
     val allowedTheme = getTheme(theme) match {
       case Some(t) => t.get(0) match {
-        case id: String => !disallowedThemes.contains(id)
-        case _ => false
+        case id: String => id match {
+            case "MISSING" => None
```
Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala]: What's this "MISSING"?

File: src/test/scala/com/mozilla/telemetry/MainSummaryViewTest.scala [hunk 2]
```diff
@@ -1790,16 +1790,54 @@ class MainSummaryViewTest extends FlatSpec with Matchers{
...
     )
 
-    val json3theme = parse("""{"id": "okayid"}""")
+    val json3theme = parse("""{"id": "firefox-compact-light@mozilla.org"}""")
 
-    MainSummaryView.getQuantumReady(json3e10s, json3addons, json3theme) should be (false)
+    MainSummaryView.getQuantumReady(json3e10s, json3addons, json3theme) should be (Some(false))
 
-    // not quantum ready with old-style theme
+    // not quantum-ready with old-style addon
```
Comment (maurodoglio) [inline src/test/scala/com/mozilla/telemetry/MainSummaryViewTest.scala]: s/addon/theme

File: src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala (hunk 1 above)
Comment (fbertsch) [inline src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala]: From the `getTheme` function. Not a super big fan, if you think there's a better refactor I'm all ears.

Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala]: Why doesn't getTheme return an Option? If "MISSING" is the default value of a column, we should probably set its value to theme.getOrElse("MISSING").

**Comment by fbertsch** [comment]
@maurodoglio fixed the existing isQuantumReady definition, and changed all `GC_MAX_PAUSE` to `GC_MAX_PAUSE_2`. RFAL

File: src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala (hunk 1 above)
Comment (maurodoglio) [inline src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala]: discussed this offline, it's not something we should address in this PR

**Pull request merged**